cfe_internal/update/update_policy.cf

```diff
@@ -42,7 +42,7 @@ bundle agent cfe_internal_update_policy
 
       # Look for a bundle that matches what the user wants
       "found_matching_user_specified_bundle"
-        slist => bundlesmatching( "$(def.mpf_update_policy_bundle)" );
+        slist => bundlesmatching( "^$(def.mpf_update_policy_bundle)$" );
```

Comment:
This has no effect because `bundlesmatching()` is anchored. I suppose it is good for clarity though. :+1: